app/build.gradle

```diff
@@ -136,10 +136,11 @@ dependencies {
     compile "com.android.support.test.espresso:espresso-idling-resource:$rootProject.espressoVersion"
 
     //Play Services
+    compile "com.google.android.gms:play-services-auth:$rootProject.playServicesVersion"
```

Comment:
The piece of code that we have in Settings it's not working anymore. The class was removed. I first deleted that and realized that I can just add this dependency.  The reason was the upgrade of the gms version.